### PR TITLE
Bring up to date with latest i18nProvider method, updates translations

### PR DESCRIPTION
**Pull request by joshf**
as per https://github.com/marmelab/react-admin/pull/5245

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
+
+const spanishMessages: TranslationMessages = {
   ra: {
     action: {
       add: "Añadir",
       add_filter: "Añadir filtro",
       back: "Ir atrás",
-      bulk_actions: "1 item seleccionado |||| %{smart_count} items selecccionados",
+      bulk_actions:
+        "1 item seleccionado |||| %{smart_count} items selecccionados",
```
Comment (manutuero) [inline src/index.ts]: Can you fix this word `selecccionados` by `seleccionados` please? Was bad written by us.

**Review by manutuero — CHANGES_REQUESTED** [review]
Thanks for collaborate! :)

File: src/index.ts (hunk 1 above)
Comment (joshf) [reply src/index.ts]: done!

**Review by manutuero — APPROVED** [review]
Thanks for adding Typescript !

**Pull request merged**